src/Datasource/ConnectionManager.php

```diff
@@ -30,7 +30,7 @@ class ConnectionManager
 {
 
     use StaticConfigTrait {
-        config as protected _config;
```

Comment:
Do we need to keep the _config around for BC reasons (if as getter or setter we dont know probably)